identity/opendj/opendj_backendprop.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Werner Dijkerman (ikben@werner-dijkerman.nl)
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+#
+
+DOCUMENTATION = '''
+---
+module: opendj_backendprop
+short_description: Will update the backend configuration of OpenDJ via the dsconfig set-backend-prop command.
+description:
+   - This module will update settings for OpenDJ with the command set-backend-prop.
+   - It will check first via de get-backend-prop if configuration needs to be applied.
+version_added: "2.2"
+author:
+    - Werner Dijkerman
+options:
+    opendj_bindir:
+        description:
+            - The path to the bin directory of OpenDJ.
+        required: false
+        default: /opt/opendj/bin
+    hostname:
+        description:
+            - The hostname of the OpenDJ server.
+        required: true
+    port:
+        description:
+            - The Admin port on which the OpenDJ instance is available.
+        required: true
+    username:
+        description:
+            - The username to connect to.
+        required: false
+        default: cn=Directory Manager
+    password:
+        description:
+            - The password for the cn=Directory Manager user.
+            - Either password or passwordfile is needed.
+        required: false
+    passwordfile:
+        description:
+            - Location to the password file which holds the password for the cn=Directory Manager user.
+            - Either password or passwordfile is needed.
+        required: false
+    backend:
+        description:
+            - The name of the backend on which the property needs to be updated.
+        required: true
+    name:
+        description:
+            - The configuration setting to update.
+        required: true
+    value:
+        description:
+            - The value for the configuration item.
+        required: true
+    state:
+        description:
+            - If configuration needs to be added/updated
+        required: false
+        default: "present"
+'''
+
+EXAMPLES = '''
+  - name: "Add or update OpenDJ backend properties"
+    action: opendj_backendprop
```

Comment:
Is their a reason you are using `action:` rather than calling the module directly?
```opendj_backendprop:```